scripts/us_eia/opendata/process/coal.py

```diff
@@ -84,6 +84,7 @@ def extract_place_statvar(series_id, counters):
         measure = m.group(1)
         if measure.startswith("SHIP"):
             # TODO: model destination / source port as well
+            print(series_id)
```

Comment:
Please remove this print statement if it's only used for debugging. Alternatively, use the logging module to log this.

Reply:
we are not doing coal as now it's on hold I will remove this file